bridges/testing/environments/rococo-westend/bridges_rococo_westend.sh

```diff
@@ -250,6 +251,21 @@ case "$1" in
           "$GLOBAL_CONSENSUS_WESTEND_SOVEREIGN_ACCOUNT" \
           10000000000 \
           true
+      # create foreign asset pool
+      create_pool \
```

Comment:
@antkve Anthony, cool, create_pool / add_liquidity works, but I think we need here more fixes for polkadot-sdk, please port the changes from here https://github.com/polkadot-fellows/runtimes/pull/764 to the polkadot-sdk